unitTests/MockElement.ts

```diff
@@ -0,0 +1,39 @@
+import { mock } from './MockEnvironment';
+
+export type EventListenersMap = { [type: string]: ((event: Event) => any)[] };
+
+export type EventTriggersMap = { [type: string]: Event[] };
```

Comment:
Another option here would be `Record<string, Event[]>`.

Reply:
Ah yes, that looks cleaner. I'll use `Record<string, Event[]>` and `Record<string, ((event: Event) => any)[]`.